### DISCUSSION TITLE
Does PnP Search work with Graph Connectors?

### DISCUSSION THREAD

**Discussion by ITProCorner** [microsoft-search/pnp-modern-search#2449] [Q&A]
Can you guys provide some guidance on configuring verticals for ServiceNow Graph Connector or Enterprise websites Graph Connector?
Does PnP Search provide support for these, can PnP Search work with Viva Topics as well?

**Comment by wobba** [comment]
Graph Connectors work. See https://microsoft-search.github.io/pnp-modern-search/usage/search-results/data-sources And also see official Graph Search documentation on how to query external data sources - https://learn.microsoft.com/en-us/graph/api/resources/searchrequest (entityTypes).

Viva Topics are currently not supported in the Graph Search API, so cannot be supported in these web parts.
